Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -758,8 +758,15 @@ protected static function ensureCLISubrequestsUseCurrentlyRunningPhpBinary($phpB
             return;
         }
 
-        // Resolve any symlinks that the configured php might be pointing to
-        $configuredPhpBinaryPathAndFilename = realpath($phpBinaryPathAndFilename);
+        // Try to resolve which binary file PHP is pointing to
+        exec($phpBinaryPathAndFilename . ' -r "echo PHP_BINARY;"', $output, $result);
```

Comment:
What happens if `$phpBinaryPathAndFilename` is empty?

Reply:
Sorry for the delay on getting back to you! The command would fail, and the next condition would not be entered (so it would be trying to resolve a symlink). But in any case, it is not pretty so I improved the code to avoid this.

Thanks for checking this!